src/plugins/home/public/plugin.ts

```diff
@@ -134,10 +134,10 @@ export class HomePublicPlugin
     featureCatalogue.register({
       id: 'home_tutorial_directory',
       title: i18n.translate('home.tutorialDirectory.featureCatalogueTitle', {
-        defaultMessage: 'Add data',
+        defaultMessage: 'Add sample data',
       }),
       description: i18n.translate('home.tutorialDirectory.featureCatalogueDescription', {
-        defaultMessage: 'Ingest data from popular apps and services.',
```

Comment:
NIT: I think the recommended text was 'Get started with sample data, visualizations, and dashboards'

https://github.com/opensearch-project/OpenSearch-Dashboards/issues/426#issuecomment-857019193

Reply:
Will update this text.